### PR TITLE
add credential resource to allow configure CA during onboard bridge the device

### PR DESCRIPTION
**Pull request by jkralik**

### CODE REVIEW

File: bridge/resources/secure/credential/resource.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+type Manager interface {
+	Get(req *net.Request) (*pool.Message, error)
+	Post(req *net.Request) (*pool.Message, error)
+}
+
+func New(uri string, m Manager) *Resource {
+	d := &Resource{}
+	d.Resource = resources.NewResource(uri, m.Get, m.Post, []string{credential.ResourceType}, []string{interfaces.OC_IF_BASELINE, interfaces.OC_IF_RW})
+	// don't publish cloud resource to cloud
```
Comment (Danielius1922) [inline bridge/resources/secure/credential/resource.go:41]: credential resource

Comment (jkralik) [reply bridge/resources/secure/credential/resource.go]: fixed

File: bridge/device/credential/manager.go [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+func (m *Manager) GetCAPool() []*x509.Certificate {
+	certs := make([]*x509.Certificate, 0, m.credentials.Length())
+	m.credentials.Range(func(key int, value credential.Credential) bool {
+		if value.Type != credential.CredentialType_ASYMMETRIC_SIGNING_WITH_CERTIFICATE {
+			return true
+		}
+		if value.Usage != credential.CredentialUsage_TRUST_CA && value.Usage != credential.CredentialUsage_MFG_TRUST_CA {
+			return true
+		}
+		cas, err := security.ParseX509FromPEM(value.PublicData.Data())
```
Comment (Danielius1922) [inline bridge/device/credential/manager.go:61]: equivalent function should be available in pkg/x509

Comment (jkralik) [reply bridge/device/credential/manager.go]: fixed

**Review by Danielius1922 — APPROVED** [review]

**Pull request merged**